contents/blog/company-survey-h1-2024.md

```diff
@@ -0,0 +1,99 @@
+
+We ran this survey using the Pulse Surveys app in Slack. Our previous three surveys were quarterly, but this time we waited six months. We’ll stick with bi-annual surveys moving forward.
+
+### Highest scores
+
+* **97% positive:** I understand PostHog’s goals and can see how my work contributes to them.
+* **97% positive:** PostHog is in a position to really succeed over the next three years.
+* **97% positive:** My manager and team around me genuinely care about my wellbeing. 
+
+### Lowest scores
+
+* **82% positive:** The support I am receiving and processes we have in place allow me to do my best possible work.
+* **79% positive:** I believe that my total compensation (salary + equity + benefits) is fair, relative to similar roles at other companies. 
+* **79% positive:** I see myself still working at PostHog in two years' time. 
+
+### Qualitative questions
+
+**What motivates you right now?** The most common themes people highlighted this time were the same as last.
+
+* **Team.** People used phrases such as: the team, great colleagues, helping the team, the great folks at PostHog at all levels, and improving things for teammates.
+
+* **Growth.** People used phrases such as: growing the company, seeing revenue going up, business growing, and numbers going in the right direction.
+
+* **Opportunities.** People used phrases such as: many learning opportunities, ability to be heard and try new things, so many opportunities lie ahead of us, and building a great product.
+
+**If you were to leave PostHog, what would be the reason?** There were a variety of answers to this question, all of them hypothetical, but all to be taken seriously.
+
+* **Career growth.** People used phrases such as: start my own company, getting back into management, a better opportunity for my career, a career growth move, and starting my own startup.
+
+* **The work.** People used phrases such as: not feeling like I'm making an impact, getting overwhelmed with non-fun work, stuck too long on challenging, repeating things, and burnout.
+
+* **Company stops growing.** People used phrases such as: business growth stalling, growth stalling or the company failing, and the company isn't growing anymore.
+
+### Highlights from comments
+
+These specific but anonymous quotes are broadly reflective of responses from multiple people. We’ve split them into two categories, positive and constructive, as they cover a wide range of topics.
+
+Positive:
+
+* Bouncing wacky ideas off of peeps, then building them at warp speed
+* Everyone's motivation to grow and see the company succeed.
+* Shipping a product people love and use
+* I feel bad for people who don't get to work somewhere like PostHog
+* I would say PostHog is a place where people who can work independently and without much process and support do best, but for me this 'light-touch' process works
+* All-hands is great!
```

Comment:
You can probably just have the second all hands point only, as they make the same point